index.bs

```diff
@@ -3468,33 +3468,32 @@ The W3C Recommendation Track</h3>
 		<li>Publication as a [=W3C Recommendation=].
 	</ol>
 
-	A <dfn export lt="W3C Recommendation Track document | REC Track document | Recommendation Track document | Recommendation-track document">W3C Recommendation Track document</dfn>
-	is any document whose current status is one of the four in the numbered list above.
-
 	<figure>
 		<pre class=include-raw>
 			path: basic-rec-track.svg
 		</pre>
 		<figcaption>Non-Normative Summary of the [=Recommendation Track=]</figcaption>
 	</figure>
 
-	This Process defines certain [=Recommendation Track=] publications as <dfn>Patent Review Drafts</dfn>.
-	Under the 2004 Patent Policy (and its 2017 update) [[!PATENT-POLICY-2004]],
-	these correspond to “Last Call Working Draft” in the Patent Policy;
-	Starting from the 2020 Patent Policy [[!PATENT-POLICY-2020]],
-	these correspond to “Patent Review Draft” in the Patent Policy [[!PATENT-POLICY]].
-
-	W3C <em class="rfc2119">may</em> <a href="#tr-end">end work on a technical report</a> at any time.
-
+	Only sufficiently technically mature work should be advanced.
 	As described in [[#transition-reqs]],
 	the [=Team=] will decline a request to advance in maturity stage
 	and return the specification to a [=Working Group=] for further work
 	if it determines that the requirements for advancement
 	have not been met.
 
+	Note: Should faster advancement to meet scheduling considerations be desired,
+	this can be achieved by reducing the scope of the technical report to a subset that is adequately mature and deferring
+	less stable features to other technical reports.
+
+	W3C <em class="rfc2119">may</em> <a href="#tr-end">end work on a technical report</a> at any time.
+
 <h4 id="maturity-stages">
 Maturity Stages on the Recommendation Track</h4>
 
+	A <dfn export lt="W3C Recommendation Track document | REC Track document | Recommendation Track document | Recommendation-track document">W3C Recommendation Track document</dfn>
+	is any document whose current status is one of those described below.
```

Comment:
The Note track statuses are also "below" this text, so consider

```suggestion
	is any document whose current status is one of those described in this section.
```

It would take a hostile reading to actually misunderstand this, but we have had a hostile reader.